docs/self-managed/modeler/web-modeler/troubleshooting/troubleshoot-zeebe-connection.md

```diff
@@ -23,19 +23,19 @@ understand the issue.
 
 ## Secure connection to Zeebe fails
 
-If you provide a cluster endpoint URL starting with `https`, Web Modeler will try to establish a secure connection to
+If you provide a cluster URL starting with `https`, Web Modeler will try to establish a secure connection to
 the Zeebe instance.
 In the process, it strictly validates the server's Application-Layer Protocol Negotiation (ALPN) support and its certificates
 presented against well-known certificate authorities.
 Failure to connect may have several reasons:
 
-### Configure the endpoint to accept secure connections
+### Configure the cluster URL to accept secure connections
```

Comment:
The URL is not configured here. Maybe its the `zeebe-gateway` that needs this secure configuration?

@Zelldon probably knows the best term.

Reply:
Yeah I guess what was meant here was the gateway. So like: "Configure the gateway to accept secure connections", which does by default btw

Reply:
But I guess this depends on the set up if you have an ingress in between might be something you also need to handle differently.

Reply:
> But I guess this depends on the set up if you have an ingress in between might be something you also need to handle differently.

Is this something worth noting in the documentation @Zelldon? If so, how would you go about phrasing this?